### PR TITLE
docs: add contact us form for sales; improve enterprise page

### PR DESCRIPTION
**Pull request by sharkymark**
This PR:

1. Replaces the mailto:sales@coder.com with the coder.com contact form
2. Improves the Enterprise page with a table and categories; designate OIDC, Git Auth as also limited in OSS
3. Adds TTL flag to templates page

### CODE REVIEW

File: docs/admin/upgrade.md [hunk 1]
```diff
@@ -22,10 +22,11 @@ curl -L https://coder.com/install.sh | sh
 ```
 
 The script will unpack the new `coder` binary version over the one currently installed.
-Next, you can restart Coder with the following command (if running it as a system
+Next, you can restart Coder with the following commands (if running it as a system
 service):
 
 ```console
+systemctl daemon-reload
```
Comment (matifali) [inline docs/admin/upgrade.md:29]: Why is daemon-reload required?

Comment (ghuntley) [reply docs/admin/upgrade.md:29]: If upgrading `coder` via the curl to bash.

**Review by ghuntley — APPROVED** [review]

**Comment by sharkymark** [comment]
@ghuntley @kylecarbs was going to add the code changes to sales@coder.com in this PR. @kylecarbs how do you want to handle this?

**Comment by kylecarbs** [comment]
It'd be better if @ghuntley took care of those cases if possible

**Pull request merged**